packages/frontend-2/components/dashboard/Sidebar.vue

```diff
@@ -0,0 +1,178 @@
+<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
```

Comment:
On mobile, the sidebar comes in from the side, over the top of the content. It will not fill the full width, instead a div with backdrop-blur is added between the menu and the main content. On mobile, you would expect to be able to tap this and close the menu, which I have done. 

I added v-keyboard-clickable, but it didn't stop flagging the usual eslint complaints. This item isn't even visible on devices with keyboards, so i just added these.